### DISCUSSION TITLE
About docker image

### DISCUSSION THREAD

**Discussion by 1998729** [jertel/elastalert2#1052] [Q&A] [answered]
Which image version currently supports es6.8? I'm currently using latest and it doesn't seem to support 6.8

**Comment by 1998729** [comment]
<img width="597" alt="image" src="https://user-images.githubusercontent.com/88186802/208038389-3f597f9f-3d62-4290-b860-f6602e199e24.png">

**Comment by 1998729** [comment]
@jertel

**Comment by 1998729** [comment, marked as answer]
see: https://hub.docker.com/r/jertel/elastalert2/tags

**Reply by 1998729** [reply]
Please help to confirm which version supports 6.X

**Reply by nsano-rururu** [reply]
We're dropping the es6 support code in version 2.4.0. Use an older version of the docker image.
https://github.com/jertel/elastalert2/blob/master/CHANGELOG.md#240

**Reply by 1998729** [reply]
> We're dropping the es6 support code in version 2.4.0. Use an older version of the docker image. https://github.com/jertel/elastalert2/blob/master/CHANGELOG.md#240

grateful! I'm going to verify version 2.3.0
